.github/requirements.txt

```diff
@@ -9,3 +9,4 @@ jsonlines
 tensorflow-cpu
 kubernetes
 pyarrow
+apache-airflow-providers-google
```

Comment:
is this needed for github runners?

Reply:
discussed offline

Reply:
Yes, otherwise some tests fail with dependency error